settings.gradle.kts

```diff
@@ -37,6 +37,7 @@ sequenceOf(
   "extra-kotlin",
   "key",
   "nbt",
+  "properties",
```

Comment:
Let's add this module to the bom just for completeness sake -- feels like the kind of thing that could cause subtle funky issues down the road if versions are mismatched.